salt/_modules/metalk8s_kubernetes.py

```diff
@@ -447,6 +447,46 @@ def namespaces(**kwargs):
         _cleanup(**cfg)
 
 
+def namespace_add_annotation(
+        name,
+        annotation_key,
+        annotation_value,
+        **kwargs):
+    '''
+    Patch the Kube-system namespace with MetalK8s version
```

Comment:
Not only with the MetalK8s version, given you pass in key and value, right?

Reply:
Well, the key:value pair can be anything... And yes I can make the message more generic